third_party/blink/renderer/modules/cobalt/h5vcc_runtime/h_5_vcc_runtime.idl

```diff
@@ -19,13 +19,13 @@
     SecureContext
 ]
 interface H5vccRuntime : EventTarget {
-    // A synchronous version of the initialDeepLink API was provided for
+    // A synchronous version of the getInitialDeepLink API was provided for
     // ease-of-use in the clients during the initial migration from Cobalt 25.
     // Asynchronous APIs are preferred here as they offload work and won't
     // affect Cobalt performance as much.
     // We mark the synchronous APIs as nullable to signify we may deprecate
     // them in the future in favor of the asynchronous equivalents.
-    readonly attribute DOMString? initialDeepLink;
+    DOMString? getInitialDeepLinkSync();
```

Comment:
This seems to support the ["Accessors should behave like properties, not methods"](
https://www.w3.org/TR/design-principles/#attributes-vs-methods), in particular "A getter must not have any (observable) side effects. If you have expected side effects, use a method." Is that reason for the change? The CL description doesn't give any information on the why nor the what.

Also can we add WPT-internal tests for this now?